### PR TITLE
Bug 793722 windows 7-10 screenshot

### PR DESCRIPTION
**Pull request by gileli121**
integrate window capture.exe with screenshot-win32.exe

This update fixes the bug
https://bugzilla.gnome.org/show_bug.cgi?id=793722

Using an external capture script that can be found here:
https://github.com/gileli121/capture-window

We use this capture script to capture single window that is out off the screen or covered by another window. this path does not require to move the window to be fully visible on the monitor and not covered by another window. the window will captured without any need of that. working also on microsoft edge and any window.

### CODE REVIEW

File: plug-ins/screenshot/screenshot-win32.c [hunk 1]
```diff
@@ -427,86 +448,121 @@ primDoWindowCapture (HDC  hdcWindow,
...
   /* Get the device context for the whole screen
-   * even if we just want to capture a window.
-   * this will allow to capture applications that
-   * don't render to their main window's device
-   * context (e.g. browsers).
+  * even if we just want to capture a window.
+  * this will allow to capture applications that
+  * don't render to their main window's device
+  * context (e.g. browsers).
   */
```
Comment (gileli121) [inline plug-ins/screenshot/screenshot-win32.c]: This comment is no longer relevant..

File: .gitignore [hunk 2]
```diff
@@ -50,3 +50,4 @@
 /stamp-h.in
 /stamp-h1
 /test-driver
+plug-ins/screenshot/screenshot - Shortcut.lnk
```
Comment (gileli121) [inline .gitignore]: This line should be deleted

**Comment by luzpaz** [comment]
@gileli121 unfortunately the gimp devs don't prefer to use github to submit PRs/patches. Better results would be to use bugzilla. Eventually they will be moving to gitlab but for the time being bugzilla is where they operate. Do you want me to open a ticket for you ?

**Comment by gileli121** [comment]
Hello, I opened this pull request by mistake.You can close this pull request. I finally developed a better patch that does not require an extra exe file to perform the action (of capturing window)

I uploaded the patch as an attachment zip file to here:
https://bugzilla.gnome.org/show_bug.cgi?id=793722

But I still not received a response about my patch. I do not know why I have not received a response yet.
I guess I'll have to wait a while until I get a response.

**Comment by Jehan** [comment]
Yep closing this one.

As for the bug report, don't worry, we will review as soon as someone can make the time for it. :-)

**Pull request closed without merging**